Sources/Interaction/Widgets/LineWidget/index.js

```diff
@@ -245,11 +238,13 @@ function vtkLineWidget(publicAPI, model) {
 
     publicAPI.invokeInteractionEvent();
     publicAPI.render();
+
+    return EVENT_ABORT;
```

Comment:
@floryst @jourdain would you know if i'm missing a `return VOID` somewhere in `LineWidget.moveAction` ?

Reply:
The VOID is just for the linter in case of multiple return path. As long as you don't intent to ABORT, you are good regardless of what you return.

Reply:
That's what I'm wondering: shouldn't there be a case in LineWidget.moveAction where we should not abort? Like [here](https://github.com/Kitware/vtk-js/pull/549/files#diff-99b12f1ff9fb5fe232106e92d9054025R159) for HandleWidget.

Reply:
@martinken

Reply:
I think we should only abort when the widgetState is not START.

Reply:
Should we still call `invokeInteractionEvent` and `render` if START? We only do render for the handlewidget: see https://github.com/agirault/vtk-js/blob/63554fbfe28c7877c28a2e6a8dfaa97ab239b528/Sources/Interaction/Widgets/HandleWidget/index.js#L150-L159

Reply:
Testing things now.